src/types/venues.ts

```diff
@@ -156,6 +229,7 @@ export const urlFromImage = (
     : defaultValue;
 };
 
+// TODO: refactor this somewhere more useful, maybe utils?
 export const createJazzbar = (values: FormValues): Venue => {
```

Comment:
> refactor this somewhere more useful, maybe utils?

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1080/commits/58906691ed0baa57fa53bf359d66839f100e6c5d